site/content/en/docs/adopters/index.md

```diff
@@ -29,6 +29,7 @@ If you are using Kueue, feel free to open a pull request to add your organizatio
 |       [Onna Technologies, Inc](https://onna.com)        | End User |              Unstructured Data Management Platform              |            batch/job </br>            |     [@gitcarbs](https://github.com/gitcarbs)     |
 |       [IBM Research](https://research.ibm.com)          | End User | Part of [MLBatch][mlbatch] stack for running AI/ML workloads    | AppWrapper</br>PyTorchJob</br>RayJob  |    [dgrove-oss](https://github.com/dgrove-oss)   |
 |       [Innovatrics](https://www.innovatrics.com/)       | End User |                     On-premise ML Platform                      |          batch/job </br> pod          |    [@mmolisch](https://github.com/mmolisch)      |
+| [Red Hat, Inc.](https://www.redhat.com/en) | End User | Cloud/On-premise ML Platform | Ray Cluster <br> ... | [@varshaprasad96](https://github.com/varshaprasad96) |
```

Comment:
I think we use more integrations than just "Ray Cluster".

https://github.com/opendatahub-io/kueue/blob/dev/config/components/manager/controller_manager_config.yaml#L34

Reply:
I added more integrations but at risk of bloating the page with them I chose to use the ellipsis to indicate we support more. Unless it is a requirement to state all supported integrations?

Reply:
Ah I see. I missed the ellipsis.